openstack/networking/v2/subnets/requests.go

```diff
@@ -79,7 +79,7 @@ type CreateOpts struct {
 	Name            string                `json:"name,omitempty"`
 	TenantID        string                `json:"tenant_id,omitempty"`
 	AllocationPools []AllocationPool      `json:"allocation_pools,omitempty"`
-	GatewayIP       *string               `json:"gateway_ip"`
+	GatewayIP       *string               `json:"-"`
```

Comment:
Cool - thank you for the attention on this. :)

I think the proposed workflow is good. This mirrors the default behavior in the Neutron CLI tool.